src/code-generator/test/non-interactive-cli.test.ts

```diff
@@ -10,56 +10,123 @@ beforeEach(async () => {
 });
 
 afterEach(async () => {
-  await fsExtra.remove(emptyFolderForATest);
+  //await fsExtra.remove(emptyFolderForATest);
 });
 
 describe("Non-interactive CLI component tests", () => {
-  test("When installing with specified app name, then the created folder has this specific name", async () => {
-    // Arrange
-    const specificAppName = "test";
+  describe("ORM type", () => {
+    test("When ORM type is Prisma, then the created DAL folder has prisma dependency and files", async () => {
+      // Arrange
 
-    // Act
-    await execa("ts-node", [
-      "./bin/cli.ts",
-      "immediate",
-      `--target-directory=${emptyFolderForATest}`,
-      `--app-name=${specificAppName}`,
-    ]);
+      // Act
+      await execa("ts-node", [
+        "./bin/cli.ts",
+        "immediate",
+        `--target-directory=${emptyFolderForATest}`,
+        `--app-name=test`,
+        "--orm=prisma",
+      ]);
```

Comment:
Could you please extract this to test helper under `runImmidiate` with argument for what option to run.

in case we would change how we run the CLI tool we would only need in one place

Reply:
Sure?

A classic testing discussion about 'copy' vs 'reuse'